index.html

```diff
@@ -276,7 +283,7 @@ <h2>Data Model</h2>
       </p>
 
       <section>
-        <h3>RevocationList2021Status</h3>
+        <h3>StatusListEntry2021</h3>
```

Comment:
Here and throughout:

```suggestion
        <h3>StatusList2021Entry</h3>
```

Reply:
Same comment as https://github.com/w3c-ccg/vc-status-list-2021/pull/24/files#r825341914.

Reply:
Fixed in 01a681de2d2c2b91fc13d5ed6b272ea27254abe4.